CONVERTER-SERVICE/src/app.js

```diff
@@ -13,10 +13,17 @@ with Jalasoft.
 
 const express = require('express');
 const dotenv = require('dotenv');
-
+const Database = require('./database/database.db');
+require('./models/file.model');
 const app = express();
 dotenv.config();
 
+const database = new Database(process.env.MONGO_URI, process.env.MONGO_DB);
+
+database.connectDatabase()
+  .then(() => console.info('Database is connected'))
```

Comment:
We shouldn't use console.info or console.log in the final code.
And the structure of .then and .chatch we are using is in block not inline.

Reply:
It's already done